tools/c7n_oci/tests/oci_common.py

```diff
@@ -56,6 +56,9 @@ def fetch_validation_data(self, resource_manager, operation, resource_id):
         else:
             return oci.util.to_dict(resources.data)
 
+    def addCleanup(self, func, *args, **kw):
+        pass
```

Comment:
why is this needed?, having a null implementation here is likely to confuse callers

Reply:
The following line in test_instance.py test_add_defined_tag_to_instance (line 47) will raise a NotImplementedError otherwise. 
```python
output = self.capture_logging(name=policy.resource_manager.log.name, level=logging.DEBUG)
``` 
Let me know if you want me to add a clarifying comment on the same

Reply:
this method does need an implementation, I think its because we're using the wrong base test class, PyTestUtils is a better base that implements this method, the reason for this distinction is because we also have unittest.TestCase classes that use a different cleanup implementation.